dtc-aws-plugin/package.json

```
@@ -22,10 +22,10 @@
     "@aws-sdk/client-sns": "^3.645.0",
     "@aws-sdk/lib-dynamodb": "^3.645.0",
     "@aws-sdk/util-dynamodb": "^3.645.0",
-    "@cgauge/assert": "^0.18.1",
-    "@cgauge/dtc": "^0.18.1",
-    "@cgauge/nock-aws": "^0.18.1",
-    "@cgauge/type-guard": "^0.18.1"
+    "@cgauge/assert": "^v0.19.0",
```

There is no `v` in the versions

```suggestion
    "@cgauge/assert": "^0.19.0",
```